meta/main.yml

```diff
@@ -3,9 +3,9 @@ galaxy_info:
   author: idealista
   company: Idealista S.A.U
   description: Opengrok role
-  min_ansible_version: 2.7.0
+  min_ansible_version: 2.9.0
   license: Apache 2.0
   platforms:
     - name: Debian
       versions:
-        - stretch
```

Comment:
I think we could keep both versions, as we run the tests on both Debian versions